net/keepalived/Makefile

```diff
@@ -29,7 +29,13 @@ define Package/keepalived
   CATEGORY:=Network
   TITLE:=Failover and monitoring daemon for LVS clusters
   URL:=http://www.keepalived.org/
-  DEPENDS:=+PACKAGE_libnl-genl:libnl-genl +libopenssl +libip4tc +IPV6:libip6tc +libxtables
+  DEPENDS:= \
+    +PACKAGE_libnl-genl:libnl-genl \
+    +libopenssl \
+    +libip4tc \
+    +IPV6:libip6tc \
+    +libxtables \
+    +kmod-macvlan
```

Comment:
As a user I'd prefer to enable this kmod package explicitly when I need to use vmac feature (not all users of keepalived will necessarily be configuring vmac)

On the other side, the warnings generated don't make it obvious whats wrong (i.e. macvlan module missing). 

Thoughts?

Reply:
I am using macvlan and it took me some time to see the warning and realize whats wrong.
I think we should add this to the DEPENDS list because a user will not know all internals and dependency.
If I configure keepalived with the options it provides it should works as expected (good user experience)